app/controllers/groupings_controller.rb

```diff
@@ -34,14 +51,23 @@ def update
     redirect_to edit_grouping_path
   end
 
+  def destroy
+    @grouping.destroy
+    redirect_to groupings_path
```

Comment:
Should we show an error whenever something goes wrong while destroying the record?